animareflection/ui
# Image Components (Universal + Next.js Enhanced)

## Description

##### Task link: https://trello.com/c/IGLe9Whv/112-image-component

Added `Image` components for both the `universal` module and `next` module. Update storybook builder to use Next.js by default.

## Test Steps

1) Verify that components render appropriately in storybook and a downstream app.
2) Verify that all tests pass.
3) Verify that new builder doesn't cause any breaking changes that aren't accounted for.
4) Ensure that code quality is up to par.
